.github/workflows/wheels.yml

```diff
@@ -212,12 +213,53 @@ jobs:
           echo "CIBW_BUILD_FRONTEND=$CIBW" >> "$GITHUB_ENV"
 
       - name: Build wheels
+        if: matrix.buildplat[1] != 'win' && matrix.buildplat[2] != 'ARM64' # Skip for win_arm64 for manual build
         uses: pypa/cibuildwheel@cf078b0954f3fd08b8445a7bf2c3fb83ab3bb971 # v3.0.0b4
         env:
           CIBW_BUILD: ${{ matrix.python[0] }}-${{ matrix.buildplat[1] }}*
           CIBW_ARCHS: ${{ matrix.buildplat[2] }}
           CIBW_ENABLE: cpython-freethreading cpython-prerelease
 
+      # can be remove after https://github.com/actions/partner-runner-images/issues/90
+      - name: Install LLVM for win_arm64
+        if: matrix.buildplat[1] == 'win' && matrix.buildplat[2] == 'ARM64'
+        uses: KyleMayes/install-llvm-action@v2
+        id: install_llvm
+        with:
+          version: 20.1.6
+          force-url: https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.6/LLVM-20.1.6-woa64.exe
+          env: false # Do not set CC/CXX by the action          
+
+      # pkgconf and FortranRuntime hacks may be removed after fixes in according libraries
+      - name: Build wheel for win_arm64 manually
+        if: matrix.buildplat[1] == 'win' && matrix.buildplat[2] == 'ARM64'
+        shell: cmd
+        run: |
+          CALL "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" arm64
+
+          pip install build
+          pip install -r requirements/build.txt -r requirements/openblas.txt
+          python -c "import scipy_openblas32; print(scipy_openblas32.get_pkg_config())" > %GITHUB_WORKSPACE%\scipy-openblas.pc
+          
+          set VCPKG_ROOT=C:\vcpkg
+          vcpkg install pkgconf
```

Comment:
I assume that this is because we're still getting the x86 version from Chocolatey, and that doesn't have win-arm64? 

Once we add `win_arm64` support to https://pypi.org/project/pkgconf/, we can use that instead I think - easier to understand and should work equally well.

Reply:
Actually, it does seem to work fine for NumPy: https://github.com/numpy/numpy/pull/29039

Reply:
That said, `choco` has always been slow and a little annoying to deal with. So maybe `vcpkg` is an improvement either way. If you put it in a separate job step, which is useful anyway I think, you can see how much faster it is compared to `choco` (that takes about a minute).